api/aggregator.proto

```diff
@@ -7,50 +7,49 @@ import "google/protobuf/empty.proto";
 import "api/snark_messages.proto";
 
 service Aggregator {
-    // Fetch the verification key from the aggregator service to be able to verify the proofs
-    // on-chain (instantiate the verifier contract)
-    //
-    // This verification key corresponds to the aggregator statement (this statement
-    // is basically multiple calls to the SNARK verification routine)
+    // Fetch the verification key corresponding to the aggregator statement
+    // (the statement including multiple calls to the SNARK verification
+    // routine). This is used to verify batches of proof on chain (required to
```

Comment:
proofs

Reply:
> This is used to verify batches of proof on chain (required to instantiate the verifier contract)

-> `This is used to instantiate the Zecale verifier contract and allows to verify wrapping proofs on-chain`?

Reply:
I've reworded.  Let me know if that makes sense.  (I'd like to avoid implying that the verifier contract is the *only* use of this key, since in general anyone may want to verify a batch.)